MiniAn/minian_parameters.py

```diff
@@ -0,0 +1,335 @@
+import os
+import sys
+import inspect
+import numpy as np
+
+# Import for MiniAn lib
+import minian.utilities as mnUtils
+import minian.preprocessing as mnPreproc
+import minian.initialization as mnInit
+import minian.cnmf as mnCnmf
+import minian.motion_correction as mnMotcorr
+
+sys.path.append('..')
+import utilities as utils
+import minian_utilities as mn_utils
+import minian_definitions as mn_defs
+
+class MinianParameters:
+    '''
+    MinianParameters
+    '''
+
+    def __init__(self, danse_parameters):
+        self.paths   = danse_parameters.get(mn_defs.ParametersKeys.PATHS, {})
+        self.parameters  = danse_parameters.get(mn_defs.ParametersKeys.PARAMETERS, {})
+        self.preview = False
+        if mn_defs.ParametersKeys.PREVIEW in danse_parameters:
+            self.preview = True
+            self.preview_parameters = danse_parameters[mn_defs.ParametersKeys.PREVIEW]
+
+        os.environ["OMP_NUM_THREADS"] = "1"
+        os.environ["MKL_NUM_THREADS"] = "1"
+        os.environ["OPENBLAS_NUM_THREADS"] = "1"
+        os.environ["MINIAN_INTERMEDIATE"] = os.path.join(self.paths[mn_defs.ParametersKeys.TMP_DIR], "intermediate")
+
+        parameters = self.parameters
```

Comment:
Is it necessary?

Reply:
No. I don't remember why I did it like that I think I forgot to remove it